app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/activity_details.scala.html

```diff
@@ -0,0 +1,72 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.tariffclassificationfrontend.config.AppConfig
+@import uk.gov.hmrc.tariffclassificationfrontend.models.{BTIApplication, Case, CaseStatus, Decision, Event, StoredAttachment}
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.partials.{attachments, complete_case_button}
+@import uk.gov.hmrc.tariffclassificationfrontend.utils.Dates
+@import uk.gov.hmrc.tariffclassificationfrontend.models.{EventType, CaseStatusChange}
+
+@(c: Case, events: Seq[Event])(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+    <div id="activity_heading">
+        <h3 class="heading-medium mt-0">@messages("case.menu.activity")</h3>
+    </div>
+
+    <table class="govuk-table">
+        <thead class="govuk-table__head">
+            <tr class="govuk-table__row">
+                <th class="govuk-table__header" scope="col">Name</th>
+                <th class="govuk-table__header" scope="col">Description</th>
+                <th class="govuk-table__header" scope="col">Date</th>
+
+            </tr>
+        </thead>
+        <tbody class="govuk-table__body">
```

Comment:
I dont think these classes actually do anything unless you've added extra styles
See http://govuk-elements.herokuapp.com/data/

Reply:
Pol is fixing the styles :)